src/plugins/vis_default_editor/public/components/controls/field.tsx

```diff
@@ -52,7 +52,7 @@ function FieldParamEditor({
 }: FieldParamEditorProps) {
   const [isDirty, setIsDirty] = useState(false);
   const selectedOptions: ComboBoxGroupedOptions<IndexPatternField> = value
-    ? [{ label: value.displayName || value.name, target: value }]
+    ? [{ label: value.displayName, target: value }]
```

Comment:
:information_source: The `displayName` can now no longer be undefined (and will automatically be the `name` in case there is no setting active overwriting it.